crates/relayer-types/src/signer.rs

```diff
@@ -34,3 +40,9 @@ impl AsRef<str> for Signer {
         self.0.as_str()
     }
 }
+
+impl Default for Signer {
+    fn default() -> Self {
+        Self::dummy()
```

Comment:
Do we really want/need this `Default` instance? I would rather make it explicit in the code when we summon a dummy signer.